src/PageContent/AssignVolunteers/AssignVolunteersController.js

```diff
@@ -156,6 +155,62 @@ class AssignVolunteersController extends Component {
         }
         return true;
     }
+    
+    handleConfirmClick1(d1) {
+        let deliverers = [d1];
+        this.updateConfirmClick(deliverers);
+    }
+    
+    handleConfirmClick2(d1, d2) {
+       // let updatedTime = moment().valueOf();
+        let deliverers = [d1, d2];
+        this.updateConfirmClick(deliverers);
+    //     if (this.deliverersAreEqual(deliverers, this.state.deliveries[this.state.selectedDeliveryId].deliverers)) {
+    //         // There was no change. I shouldn't send an update request,
+    //         // I should simply say that this delivery is up to date.
+    //         this.setState(prevState => {
+    //             let deliveries = prevState.deliveries;
+    //             deliveries[this.state.selectedDeliveryId].updatedTime = updatedTime + 1;
+    //             return {
+    //                 deliveries: deliveries,  // indicate that it is up to date, despite not triggering a db event
```

Comment:
I'm a bit confused what all this commented out code is for, I would just go ahead and remove it